src/components/DuosHeader.js

```diff
@@ -130,107 +260,151 @@ class DuosHeader extends Component {
                     ])
-                  ]),
+                  ])
+                ]),
 
-                  li({}, [
-                    h(Link, { id: 'link_datasetCatalog', isRendered: isLogged, to: '/dataset_catalog' }, ['Dataset Catalog'])
-                  ]),
-                  contactUsButton, supportrequestModal
+                li({}, [
+                  h(Link, { id: 'link_datasetCatalog', isRendered: isLogged, to: '/dataset_catalog' }, ['Dataset Catalog'])
                 ]),
+                li({}, [h(Link, { id: 'link_help', to: '/FAQs' }, ['FAQs'])]),
+                contactUsButton, supportrequestModal
+              ]),
 
-                ul({ isRendered: !isLogged, className: 'navbar-public' }, [
-                  li({}, [
-                    h(Link, { id: 'link_about', className: 'navbar-duos-link', to: '/home_about' }, [
-                      div({ className: 'navbar-duos-icon-about', style: navbarDuosIcon }),
-                      span({ style: navbarDuosText }, ['About'])
-                    ])
-                  ]),
-                  li({}, [
-                    h(Link, { id: 'link_help', className: 'navbar-duos-link', to: '/FAQs' }, [
-                      div({ className: 'navbar-duos-icon-help', style: navbarDuosIcon }),
-                      span({ style: navbarDuosText }, ['FAQs'])
-                    ])
-                  ]),
-                  contactUsButton, supportrequestModal
-                ])
+              ul({ isRendered: !isLogged, className: 'navbar-public' }, [
+                li({}, [
+                  h(Link, { id: 'link_about', className: 'navbar-duos-link', to: '/home_about' }, [
+                    div({ className: 'navbar-duos-icon-about', style: navbarDuosIcon }),
+                    span({ style: navbarDuosText }, ['About'])
+                  ])
+                ]),
+                li({}, [
+                  h(Link, { id: 'link_help', className: 'navbar-duos-link', to: '/FAQs' }, [
+                    div({ className: 'navbar-duos-icon-help', style: navbarDuosIcon }),
+                    span({ style: navbarDuosText }, ['FAQs'])
+                  ])
+                ]),
+                contactUsButton, supportrequestModal
               ])
-          })
+            ])
+          ]),
+        ]),
+        //NOTE: old navbar style is heavily dependent on css styles with element specific styles
+        //Hard to make that navbar flexible with material-ui's syntax
+        //For now I will use material-ui's hidden element to selectively render the two different navbars
+        //I'll look into rewriting the large navbar on a later PR
+        h(Hidden, {lgUp: true}, [
+          div({style: {display: 'flex', flexDirection: 'row', justifyContent: 'space-between'}}, [
+            img({
+              style: duosLogoImage, src: '/images/duos_logo.svg',
+              alt: 'DUOS Logo',
+              onClick: (e) => this.goToLink('/home')
+            }),
```

Comment:
The only thing we lost here is the cursor hover style over the icon. In expanded mode, it works, but here, it's a normal arrow icon. I think that has to do with the css class name in the expanded version.